### ISSUE TITLE
Weight decay during training

### ISSUE THREAD

**Issue by jytime** [gengshan-y/VCN#11]
Hi Gengshan,

In the supplementary material of VCN, you highlight that the weight decay term in PWCNet is removed. May I ask what's the effect of removing it? Is it for faster training or performance boost?

Best Regards,
Jianyuan

**Comment by gengshan-y** [comment]
Hi Jianyuan, 

From my view, weight decay is added to regularize the model / reduce overfitting, and VCN is already "regularized" in its structure, and does not need extra regularization. As I can remember in experiments, it performs better without weight decay.

**Comment by jytime** [comment]
Thanks for your quick reply.
